bugbot/rules/performancebug.py

```diff
@@ -0,0 +1,101 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from bugbot.bugbug_utils import get_bug_ids_classification
+from bugbot.bzcleaner import BzCleaner
+from bugbot.utils import nice_round
+
+
+class PerformanceBug(BzCleaner):
+    def __init__(self):
+        super().__init__()
+        self.autofix_bugs = []
+
+    def description(self):
+        return "[Using ML] Bugs with Missing Performance Impact"
+
+    def columns(self):
+        return ["id", "summary", "confidence", "autofixed"]
+
+    def get_bz_params(self, date):
+        start_date, _ = self.get_dates(date)
+
+        params = {
+            "include_fields": ["id", "summary"],
+            "f1": "creation_ts",
+            "o1": "greaterthan",
+            "v1": start_date,
+            "f2": "keywords",
+            "o2": "nowords",
+            "v2": "perf,topperf,main-thread-io",
+            "f3": "cf_performance_impact",
+            "o3": "equals",
+            "v3": ["---"],
+        }
+
+        return params
+
+    def get_bugs(self, date="today", bug_ids=[]):
+        # Retrieve the bugs with the fields defined in get_bz_params
```

Comment:
This will not have all the fields defined in get_bz_params. Only the `id` and `summary` fields will be accessible here, even if they were listed in `include_fields`.

Reply:
@suhaibmujahid, does this mean that listing `id` and `summary` in the `includes_fields` isn't relevant since they are already listed by default? I removed them and didn't see any change in the results.

```
"include_fields": ["id", "summary"],
```

Reply:
Yes, you do not need to add them. They will be added automatically.